Cargo.toml

```
@@ -41,6 +41,7 @@ members = [
     "script-storage",
     "spell-storage",
     "particle-execution",
+    "crates/system-service-deployer"
```

How about a shorter name?

```suggestion
    "crates/system-services"
```

---

`system_services=debug` would look saner

---

Or `ss-deployer`! :upside_down_face: